src/preact/compat.js

```diff
@@ -5,23 +5,24 @@ import * as mode from '#core/mode';
 const REACT_FORWARD_SYMBOL =
```

Comment:
I added a few `any` into this file. Happy to revert and add a `// @ts-nocheck` to the top instead.

Reply:
SGTM